packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -111,20 +115,38 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                                 />
                             </React.Fragment>
                         )}
-                        {!isAccumulatorContract(contract_type) && !is_vanilla && (
-                            <ContractAuditItem
-                                id='dt_bt_label'
-                                icon={
-                                    isDigitType(contract_type) ? (
-                                        <Icon icon='IcContractTarget' size={24} />
-                                    ) : (
-                                        <Icon icon='IcContractBarrier' size={24} />
-                                    )
-                                }
-                                label={getBarrierLabel(contract_info)}
-                                value={getBarrierValue(contract_info) || ' - '}
-                            />
-                        )}
+                        {!isAccumulatorContract(contract_type) &&
+                            !is_vanilla &&
+                            !isSmartTraderContract(contract_type) && (
+                                <ContractAuditItem
+                                    id='dt_bt_label'
+                                    icon={
+                                        isDigitType(contract_type) ? (
+                                            <Icon icon='IcContractTarget' size={24} />
+                                        ) : (
+                                            <Icon icon='IcContractBarrier' size={24} />
+                                        )
+                                    }
+                                    label={getBarrierLabel(contract_info)}
+                                    value={getBarrierValue(contract_info) || ' - '}
+                                />
+                            )}
+                    </React.Fragment>
+                )}
+                {hasTwoBarriers(contract_type) && (
+                    <React.Fragment>
+                        <ContractAuditItem
+                            id='dt_bt_label_1'
+                            icon={<Icon icon='IcContractStrike' size={24} />}
+                            label={localize('High barrier')}
+                            value={high_barrier}
+                        />
+                        <ContractAuditItem
+                            id='dt_bt_label_2'
+                            icon={<Icon icon='IcContractStrike' size={24} />}
+                            label={localize('Low barrier')}
+                            value={low_barrier}
+                        />
```

Comment:
```suggestion
                        {[high_barrier, low_barrier].map((barrier, index) => <ContractAuditItem
                            id={`dt_bt_label_${index + 1}`}
                            icon={<Icon icon='IcContractStrike' size={24} />}
                            key={barrier}
                            label={high_barrier === barrier ? localize('High barrier') : localize('Low barrier')}
                            value={barrier}
                        />}
```
@kate-deriv nit-pick)

Reply:
Thank you for suggestion